### PR TITLE
Change default compression setting

### PR DESCRIPTION
**Pull request by nelio2k**
The default is Auto, instead of None.

### CODE REVIEW

File: modules/learn/pages/clusters-and-availability/xdcr-advanced-settings.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ The performance of XDCR can be fine-tuned, by means of configuration-settings, s
 | Option | Description
 
 | XDCR Compression Type
-| Defines whether documents are to be compressed for XDCR, and if so, which  compression type is to be used.The default is `None`.
+| Defines whether documents are to be compressed for XDCR, and if so, which  compression type is to be used.The default is `Auto`, which will allow compression if both source and target support it.
```
Comment (jamesnocentini) [inline modules/learn/pages/clusters-and-availability/xdcr-advanced-settings.adoc:17]:
```suggestion
| Defines whether documents are to be compressed for XDCR, and if so, which compression type is to be used.
The default is `Auto`, which will allow compression if both source and target support it.
```

**Review by jamesnocentini — CHANGES_REQUESTED** [review]

**Review by m123ky79 — APPROVED** [review]

**Comment by arod1987** [comment]
+1

**Review by arod1987 — APPROVED** [review]
+1

**Comment by tonyjhillman** [comment]
Will be handled by https://github.com/couchbase/docs-server/pull/1188, DOC-6568.

**Pull request closed without merging**